src/classes/groundplaces.ts

```diff
@@ -0,0 +1,167 @@
+import {
+  AutoComplete,
+  StopGroup,
+  StopCluster,
+  StopGroupGpuid,
+  StopClusterGpuid,
+  StopGroupProperties,
+  StopClusterProperties,
+  Filters,
+  GroundPlacesDiff,
+  GroundPlaceType,
+} from '../types';
+
+/**
+ * @description GroundPlaces business logic.
+ */
+export class GroundPlaces {
+  /**
+   * @description Returns a list of places.
+   * @param {string} query - Can be a name, a Gpuid, a unique name or other name.
+   * @param {Filters} filters - Filters with different options (StopGroup, StopCluster, Serviced, SegmentProvider).
```

Comment:
you certainly can change that later but i think you can use filters as an array of enums ex : `AutocompleteFilters[]` with `AutocompleteFilters` being an enum of al filters. but its more of an implem detail so no big deal !